src/docs/reference/modules/Workflows/README.md

```diff
@@ -85,6 +85,11 @@ Although many activities support multiple outcomes, they typically return only o
 For example, the _Send Email_ activity has two possible outcomes: "Done" and "Failed".  
 When the email was sent successfully, it yields "Done" as the outcome, and "Failed" otherwise.
 
+For activities that have an iterator outcome e.g _Content For Each_, return the outcome of the next activity back to the irrerate task for the irritation to continue either through the done outcome or via a fork.
```

Comment:
irrerate == iterate?

Reply:
Ah yes, knew my spelling was off so spell checked but then did not double check the result :/